arangod/RocksDBEngine/RocksDBTransactionState.cpp

```diff
@@ -371,17 +373,20 @@ arangodb::Result RocksDBTransactionState::internalCommit() {
                _rocksTransaction->GetNumPuts() == 0 &&
                _rocksTransaction->GetNumDeletes() == 0);
     // this is most likely the fill index case
-    rocksdb::SequenceNumber seq = _rocksTransaction->GetSnapshot()->GetSequenceNumber();
+#ifdef ARANGODB_ENABLE_MAINTAINER_MODE
+//    rocksdb::SequenceNumber seq = _rocksTransaction->GetSnapshot()->GetSequenceNumber();
```

Comment:
can we clean up this part a bit?
seems there is now some commented-out code which should be removed now.